src/frontend/komponenter/Fagsak/Vilkårsvurdering/Registeropplysninger/RegisteropplysningerTabell.tsx

```diff
@@ -85,53 +112,57 @@ const RegisteropplysningerTabell: React.FC<IRegisteropplysningerTabellProps> = (
-                                                              tom: periode.tom,
-                                                          })
-                                                }
-                                            />
-                                        </TabellRad>
-                                    );
-                                })
-                        ) : (
+                        {manglerOpplysninger && (
                             <TabellRad key={`${opplysningstype}_ukjent`}>
                                 <td children={'Ingen opplysninger'} />
                                 <td children={'-'} />
                             </TabellRad>
                         )}
+                        {!manglerOpplysninger &&
+                            synligHistorikk.map(periode => (
+                                <TabellRad key={`${periode.fom}_${periode.tom}_${periode.verdi}`}>
+                                    <td children={periode.verdi} />
+                                    <td
+                                        children={
+                                            opplysningstype === Registeropplysning.SIVILSTAND ||
+                                            opplysningstype === Registeropplysning.DØDSBOADRESSE
+                                                ? tilVisning(
+                                                      periode.fom
+                                                          ? kalenderDato(periode.fom)
+                                                          : undefined
+                                                  )
+                                                : periodeToString({
+                                                      fom: periode.fom,
+                                                      tom: periode.tom,
+                                                  })
+                                        }
+                                    />
+                                </TabellRad>
+                            ))}
                     </tbody>
                 </Tabell>
             </Container>
+            {skalVæreEkspanderbar && (
+                <HøyrejustertKnapperad>
+                    <Button
+                        variant="tertiary"
+                        size="small"
+                        onClick={() => settEkspandert(nåverdi => !nåverdi)}
+                    >
+                        {erEkspandert ? (
+                            <>
+                                Skjul
+                                <Collapse />
+                            </>
+                        ) : (
+                            <>
+                                Vis {historikk.length - GRENSE_FOR_EKSPANDERBAR_HISTORIKK} flere
+                                <Expand />
+                            </>
+                        )}
+                    </Button>
+                </HøyrejustertKnapperad>
+            )}
```

Comment:
Hvorfor må dette ligge utenfor containeren og tabellen?

Reply:
Litt fordi det semantisk ikke er en del av tabellen - og litt fordi containeren inni har flex-direction row og knappen derfor ville havnet til høyre for tabellen. Derfor lagde jeg en ny container `HøyrejustertKnapperad` etter containeren, og stylet den slik at knappen plasseres riktig